### PR TITLE
Feature/image component

### PR DESCRIPTION
**Pull request by T0ffik**

### CODE REVIEW

File: src/screens/Home.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import { render } from '../shared/dom.js';
 import { MovieCarousel } from '../components';
 import ApiProvider from '../providers/ApiProvider.js';
+import { ImageComponent } from '../components/ImageComponent/ImageComponent.js';
```
Comment (Veryyapeee) [inline src/screens/Home.js]: Remove unused import

File: src/components/ImageComponent/ImageComponent.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { observerListener } from '../../shared/observer';
+
+const template = (imageUrl, className, id) => {
```
Comment (Veryyapeee) [inline src/components/ImageComponent/ImageComponent.js:3]: I think we could connect that to the TileComponent

There is task to add onClick to tileComponent, if you can please update this task to use just this image component and that should do the trick

**Review by Veryyapeee — APPROVED** [review]

File: src/screens/Home.js (hunk 1 above)
Comment (T0ffik) [reply src/screens/Home.js]: Done

File: src/components/ImageComponent/ImageComponent.js (hunk 2 above)
Comment (T0ffik) [reply src/components/ImageComponent/ImageComponent.js:3]: Done

**Review by KonTra7cf — APPROVED** [review]

**Review by Mariusz-web — APPROVED** [review]

**Pull request merged**